pkg/v1/tkg/client/kubevip_update_helper.go

```diff
@@ -51,9 +51,29 @@ func ModifyKubeVipTimeOutAndSerialize(currentKubeVipPod *corev1.Pod, newLeaseDur
 		}
 		envVars = append(envVars, envVar)
 	}
-
+	envVar := corev1.EnvVar{Name: "cp_enable", Value: "true"}
```

Comment:
nit-ish: @aartij17 Thanks for these changes. Is it possible to move these changes into a helper function? Or maybe refactor `ModifyKubeVipTimeOutAndSerialize` to something like `ModifyKubeVipAndSerialize` and have that call meaningful helper functions?

Reply:
Makes sense! I have refactored this part a bit :) PTAL!

Reply:
Thanks!